websites/T/TruckersMP/presence.ts

```diff
@@ -1,380 +1,140 @@
-var presence = new Presence({
-  clientId: "624006279769227265" // CLIENT ID FOR YOUR PRESENCE
-});
-
-var item: any,
-  user: any,
-  item2: any,
-  item3: any,
-  server: any,
-  players: any,
-  output: any;
-
-var browsingStamp = Math.floor(Date.now() / 1000);
+const presence = new Presence({
+    clientId: "821104573329440848"
+  }),
```

Comment:
Why the clientId change? I can add assets to the main one if needed.

Reply:
blame github that it didnt show it first

Reply:
As I explained before, I didn't know it was originally yours - bad communication on my behalf. It's done anyway 🤷🏼‍♂️